t/pipe.t

```diff
@@ -1358,11 +1358,51 @@ lua pipe kill process:
 --- config
     location = /t {
         content_by_lua_block {
+            local function set_up_ngx_tmp_conf(package_path)
+                local conf = [[
+                    events {
+                        worker_connections 64;
+                    }
+                    error_log stderr error;
+                    daemon off;
+                    master_process off;
+                    worker_processes 1;
+                    http {
+                        lua_package_path "]] .. package_path .. [[";
+                        init_worker_by_lua_block {
+                            ngx.timer.at(0, function()
+                                require "ngx.pipe".spawn{"no-such-cmd"}:wait()
+                                os.exit(0)
+                            end)
+                        }
+                    }
+                ]]
+
+                assert(os.execute("mkdir -p $TEST_NGINX_HTML_DIR/logs"))
+
+                local conf_file = "$TEST_NGINX_HTML_DIR/nginx.conf"
+                local f, err = io.open(conf_file, "w")
```

Comment:
Better use Test::Nginx's `--- user_files` feature to prepare such files? Is that possible?